app/scripts/services/brackets.coffee

```diff
@@ -17,16 +17,12 @@ angular.module('edudashAppSrv').service 'bracketsSrv', ->
     else
       switch metric
 
-        when 'AVG_MARK' then switch
-          when 0 <= val < 40 then 'POOR'
-          when 40 <= val <= 60 then 'MEDIUM'
-          when 60 < val <= 100 then 'GOOD'
-          else 'UNKNOWN'
+        when 'AVG_MARK' then throw new Error "AVG_MARK shall not be bracket"
 
         when 'AVG_GPA' then switch
-          when 1 <= val <= 3 then 'POOR'
+          when val <= 3 then 'GOOD'
           when 3 < val <= 4.2 then 'MEDIUM'
-          when 4.2 < val <= 5 then 'GOOD'  # what's the upper limit?
```

Comment:
Sorry, I have spoken with Mark, we don't have to validate range for min/max.
Weird :S